diff --git a/node_modules/@capacitor-firebase/authentication/Package.swift b/node_modules/@capacitor-firebase/authentication/Package.swift
index 909c7f0..df52fa7 100644
--- a/node_modules/@capacitor-firebase/authentication/Package.swift
+++ b/node_modules/@capacitor-firebase/authentication/Package.swift
@@ -11,7 +11,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "8.0.0"),
-        .package(url: "https://github.com/firebase/firebase-ios-sdk.git", .upToNextMajor(from: "12.7.0")),
+        .package(url: "https://github.com/firebase/firebase-ios-sdk.git", .upToNextMajor(from: "11.0.0")),
         .package(url: "https://github.com/google/GoogleSignIn-iOS", from: "9.0.0"),
         .package(url: "https://github.com/facebook/facebook-ios-sdk.git", from: "18.0.0")
     ],
